conda-recipes/cuml/meta.yaml

```diff
@@ -26,13 +26,13 @@ requirements:
     - setuptools
     - cython
     - cmake 3.12
-    - cudf 0.4.0.*
-    - libcuml 0.4.0.*
+    - cudf 0.5*
+    - libcuml 0.5*
```

Comment:
```suggestion
    - libcuml 0.5.*
```

Reply:
This isn't necessary I believe, if I'm reading https://conda.io/projects/conda-build/en/latest/source/package-spec.html#package-match-specifications correctly.
Also, related: https://www.python.org/dev/peps/pep-0440/